### ISSUE TITLE
Compiling files in locally

### ISSUE THREAD

**Issue by maximkgn** [jazzband/django-pipeline#538]
Should I expect assets to be compiled even with PIPELINE_ENABLED = False?
Obviously, locally I want to compile my files after each change, but I only want to compress them in production.

What is the right setup for this?

**Comment by jclgoodwin** [comment]
Yes. For local development I have `PIPELINE_ENABLED = False` and `DEBUG = True`, so my files are compiled (but not compressed) on each request, which sounds like what you want.

**Comment by maximkgn** [comment]
It is what I want, but it's not happening for me. What version are you using?
